frontend/src/features/recommendation/ScoreModeFilter.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ChoicesFilterSection } from 'src/components';
+
+import { ScoreModeEnum } from 'src/features/recommendation/RecommendationApi';
+
+interface Props {
+  value: string;
+  onChange: (value: string) => void;
+}
+
+function ScoreModeFilter(props: Props) {
+  const { t } = useTranslation();
+
+  const scoreModeChoices = {
+    [ScoreModeEnum.DEFAULT]: t('filter.scoreMode.default'),
+    [ScoreModeEnum.ALL_EQUAL]: t('filter.scoreMode.allEqual'),
+    [ScoreModeEnum.TRUSTED_ONLY]: t('filter.scoreMode.trustedOnly'),
+  };
+
+  return (
+    <ChoicesFilterSection
+      title={t('filter.scoreModeSection')}
+      choices={scoreModeChoices}
+      radio
+      {...props}
+    />
```

Comment:
Because this score mention the concept of trusted domains, I would propose to also add a link/button to the truted domain page as we do in the user profile

Reply:
The tooltip now includes a link to the "trusted domains" page (in https://github.com/tournesol-app/tournesol/pull/882/commits/2d085ed32dbc44107d351f82ff62b4638523d9b1)